appsec/src/helper/sampler.hpp

```diff
@@ -64,10 +59,20 @@ class sampler {
 
     std::optional<scope> get()
     {
+        if (service_config_->get_asm_enabled_status() ==
```

Comment:
Why do you need this at all? When is the sampler being called with ASM disabled?

Reply:
I don't think it's a massive issue but I can take it out